### ISSUE TITLE
Copy pasting Body instance

### ISSUE THREAD

**Issue by kof** [webstudio-is/webstudio#3326]
As a user I want to be able to copy/paste the entire body contents, because usually people put instances inside the body, not an extra box for copy pasting.

Problem has been so far that we don't know how to merge 2 bodies and there can be only one in the end.

Solution: ask the user in a dialog: Do you want to use the new body or keep current one? [new] [current][abort]

Now when user selects new one, we are overriding. If user picks the old one - we are inserting the contents of the new body inside the old body.

**Comment by kof** [comment]
Alternatively when copying body, we just copy the contents of the body and show toast (copied body contents, because body can not be copied)

Then there is no need for confirmation dialog and its all much simpler

**Comment by johnsicili** [comment]
Easier approach for sure and better than what is there. Downside is vars won't copy over and I place my vars almost exclusively on the body. Still would rather have any of these options above than none.

**Comment by TrySound** [comment]
alternatively we could copy whole page along with body

**Comment by kof** [comment]
> alternatively we could copy whole page along with body

We will need this in any case. The problem with copying body is that it feels very natural to copy the root node and paste it somewhere else.

Its super unnatural to have to create a Box inside body just so one can copy the entire thing. 

People will always intuitively go create a page first, then go to some template and try copy the body.

**Comment by TrySound** [comment]
> We will need this in any case. The problem with copying body is that it feels very natural to copy the root node and paste it somewhere else.

it also unnatural when paste deletes something

**Comment by kof** [comment]
> it is also unnatural when paste deletes something

Yes, we won't be deleting anything unless user explicitely agrees

**Comment by kof** [comment]
Closing in favor of https://github.com/webstudio-is/webstudio/issues/5797
